taskcluster/kinds/tox/kind.yml

```diff
@@ -95,13 +95,13 @@ tasks:
     pushapkscript:
         python-versions: [39]
         worker:
-            docker-image: {in-tree: 'pushapkscript-python{python_version}'}
+            docker-image: {in-tree: 'pushapkscript-test-py39'}
```

Comment:
Why remove this interpolation? I think we should leave it as long as we support the `python-versions` key.